### PR TITLE
Add Tsc build to vr-tests

### LINKED ISSUE #7484: apps/vr-tests should do a noEmit typescript build to ensure `npm start` is able to work
See #7483 for more context.

### PR DESCRIPTION
**Pull request by cliffkoh**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #7484 
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

We now run a tsc build which should help catch and prevent #7483.

#### Focus areas to test

(optional)

### CODE REVIEW

File: apps/vr-tests/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "version": "6.0.0",
   "private": true,
   "scripts": {
-    "build": "",
+    "build": "node ../../scripts/build.js ts --commonJsOnly",
```
Comment (cliffkoh) [inline apps/vr-tests/package.json]: @kenotron might need to be updated when we switch to `just`

File: scripts/tasks/ts.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 module.exports = function(options) {
   options = options || {};
+  options.commonjsOnly = options.commonjsOnly || process.argv.indexOf('--commonJsOnly') > -1;
```
Comment (JasonGore) [inline scripts/tasks/ts.js]: Where is this option consumed? Is it built-in?

Comment (cliffkoh) [reply scripts/tasks/ts.js]: further below in this file?

Comment (JasonGore) [reply scripts/tasks/ts.js]: Oh sorry, I thought it was a new option. 😄

**Comment by micahgodbolt** [comment]
any chance we can tweak this to build with a different version of typescript? not sure how that works in a mono repo.

**Comment by cliffkoh** [comment]
@kenotron

File: apps/vr-tests/package.json (hunk 1 above)
Comment (kenotron) [reply apps/vr-tests/package.json]: acknowledged - I'll wait until this PR is in before I merge mine #7415

**Pull request merged**